coriolis/tasks/replica_tasks.py

```diff
@@ -45,12 +45,26 @@ def _check_ensure_volumes_info_ordering(export_info, volumes_info):
 
         ordered_volumes_info.append(matching_volumes[0])
 
+    vol_info_cpy = None
+    if volumes_info:
```

Comment:
Can you please use the below utility method as it's functionally identical and we can reuse it anywhere we need it.
https://github.com/cloudbase/coriolis/blob/master/coriolis/utils.py#L528-L540

Reply:
Done